2020-09-25 08:00:12.946446: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2020-09-25 08:00:13.091332: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.545
pciBusID: 0000:0b:00.0
totalMemory: 10.76GiB freeMemory: 10.60GiB
2020-09-25 08:00:13.092047: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2020-09-25 08:00:20.908189: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-09-25 08:00:20.908702: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2020-09-25 08:00:20.913556: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2020-09-25 08:00:20.915333: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10229 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:0b:00.0, compute capability: 7.5)
Traceback (most recent call last):
  File "../exps/mter_test.py", line 190, in <module>
    result_dicts.append(experiment.train(rnn_config, labelled_data_config, training_config, info_config, run))
  File "../src/experiment.py", line 123, in train
    self.rnn.t_metrics.print(session_idx)
  File "../src/data/t_metrics.py", line 171, in print
    self.result_dict['va_s']['loss'][-1]))
OSError: [Errno 5] Input/output error
rm: write error: Input/output error
